src/createrepo_c.c

```diff
@@ -1340,30 +1415,45 @@ main(int argc, char **argv)
     GHashTableIter iter;
     gpointer key, value;
 
+    int skipped_pkgs = 0;
     g_hash_table_iter_init(&iter, user_data.nevra_table);
     while (g_hash_table_iter_next(&iter, &key, &value))
     {
         gchar *nevra = (gchar *) key;
         GArray *locations = (GArray *) value;
         if (locations->len > 1) {
-            g_warning("Package '%s' has duplicate metadata entries, only one should exist", nevra);
```

Comment:
We should keep the warning in place, A) because not all users will use the option, and B) because even users who do use the option should probably be aware of potential issues even if we're applying our own heuristics to deal with it.